.circleci/config.yml

```diff
@@ -62,14 +135,51 @@ jobs:
             git clone https://github.com/varnishcache/pkg-varnish-cache.git .
             git checkout << pipeline.parameters.pkg-commit >>
             tar cvzf debian.tar.gz debian --dereference
-            tar cvzf redhat.tar.gz redhat --dereference 
+            tar cvzf redhat.tar.gz redhat --dereference
             tar cvzf alpine.tar.gz alpine --dereference
+            mkdir -p ~/workspace
+            mv *.tar.gz ~/workspace
       - persist_to_workspace:
-          root: .
+          root: ~/workspace
           paths:
             - debian.tar.gz
             - redhat.tar.gz
             - alpine.tar.gz
+      - store_artifacts:
+          destination: ~/workspace
+          path: ~/workspace
+  x64-centos-7:
```

Comment:
wrong approach, instead, name only one job `package`, then, in the workflow, use the name attribute:
```
- package:
  name: aarch64-centos-7
  rest of the parameters
```

Reply:
This looks cool but then what would be the syntax for `requires` ?
```
- collect_packages:
          requires:
            - x64-ubuntu-xenial
            - x64-ubuntu-bionic
            - aarch64-ubuntu-xenial
            ...
```

Reply:
yes, that's the one, depends on the names that you specified for each job

Reply:
Done with https://github.com/martin-g/varnish-cache/commit/4ffeb7dbb85c4cfede0e475a75da0bc55d9b20f9!